web/themes/custom/votegov/src/sass/uswds-overrides/usa-card.scss

```diff
@@ -3,197 +3,191 @@
 @use "mixins" as *;
 
 .usa-card-group {
+
   a {
     @include u-display('block');
     @include u-text-decoration('no-underline');
     @include u-height('full');
 
-    @include at-media('tablet-lg') {
-      @include hover {
-        .usa-card__container {
-          background-color: $bg-light-cool;
-          border: 3px solid #3E79BB;
-        }
-
-        .vote-feature-card__icon-circle {
-          background-color: $base-white;
+    @include hover {
+      transition: 0.3s;
```

Comment:
animation transitions should be set on the default state of a link and not in a hover state to maintain animation easing for on and off hover.